Mathlib/SetTheory/Surreal/Basic.lean

```diff
@@ -301,6 +301,15 @@ instance : One Surreal :=
 instance : Inhabited Surreal :=
   ⟨0⟩
 
+@[simp]
+lemma mk_eq_mk {x y : PGame.{u}} {hx hy} : mk x hx = mk y hy ↔ x ≈ y := by
+  rw [mk, mk, Quotient.eq]; rfl
+
+@[simp]
+lemma mk_eq_zero {x : PGame.{u}} {hx} : mk x hx = 0 ↔ x ≈ 0 := by
+  change mk x hx = mk 0 numeric_zero ↔ x ≈ 0
+  rw [mk_eq_mk]
```

Comment:
```suggestion
@[simp]
lemma mk_eq_mk {x y : PGame.{u}} {hx hy} : mk x hx = mk y hy ↔ x ≈ y := Quotient.eq

@[simp]
lemma mk_eq_zero {x : PGame.{u}} {hx} : mk x hx = 0 ↔ x ≈ 0 := Quotient.eq
```

Reply:
Are there guidelines for when something should be `@[simp]` ? If I don't tag these as `@[simp]`, then I need to list them explicitly within `simp [...]` when I use them. In my current work, there is just one use of each of these two lemmas, and there may a few more later, but I don't anticipate a huge number.

Reply:
The `mk_le_mk` a few lines below these changes is currently `@[simp]`.

Reply:
Keeping the `@[simp]` for now.